### PR TITLE
CPDTP-250 add scheduled payable and paid as a delayed job

### PR DESCRIPTION
**Pull request by blackrat**
## Ticket and context

Declarations are marked as payable when they hit a cut-off milestone and are currently eligible.  This adds in a hard-coded schedule to trigger those when the milestone is reached. There is an offset of the delayed job running of 10 hours from the cut-off point, but the milestones will be strictly enforced by the payable routine checking dates and times of submission and declaration.

For the paid one, anything that is marked as payable will be included. since only thing that are marked as payable (either through the schedule, or some currently unknown support process) will be paid, so not timing should 

Ticket: https://dfedigital.atlassian.net/browse/CPDTP-250

### Code quality checks
- [ ] All commit messages are meaningful and true
- [ ] Added enough automated tests

### HTML Checks
- [ ] All new pages have automated accessibility checks
- [ ] All new pages have visual tests via Percy

### Gotchas
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Follow the steps from the ticket.

## External API changes

Does this change anything in our external APIs? If so, did you remember to update the CHANGELOG for Lead Providers? (docs/source/release-notes)

### CODE REVIEW

File: app/services/participant_event_paid_aggregator.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ParticipantEventPayableAggregator < ParticipantEventAggregator
```
[the remark this answers is not in the document]
Comment (kicferk1) [reply app/services/participant_event_paid_aggregator.rb:3]: Also... is this used anywhere at the moment?

File: config/locales/en.yml [hunk 2]
```diff
@@ -115,6 +115,8 @@ en:
     monthly: Payment amount monthly
     subtotal: Sub total
     period: Payment amount for period
+    make_payable: Mark records as payable
+    pay: Mark records as paid
```
[the remark this answers is not in the document]
Comment (blackrat) [reply config/locales/en.yml:119]: Not any more. We had buttons at one point. I need to do more of a cleanup.

File: app/services/record_declarations/actions/with_milestone.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RecordDeclarations
+  module Actions
+    module WithMilestone
```
[the remark this answers is not in the document]
Comment (blackrat) [reply app/services/record_declarations/actions/with_milestone.rb]: Gone.

File: app/controllers/finance/ecf/payment_breakdowns_controller.rb [hunk 4]
```diff
@@ -11,6 +11,7 @@ def show
           contract: @ecf_lead_provider.call_off_contract,
           event_type: :started,
         )
+        @type = "eligible"
```
[the remark this answers is not in the document]
Comment (blackrat) [reply app/controllers/finance/ecf/payment_breakdowns_controller.rb]: This changed many times. I've cleaned up for the latest commit. This was to control which button we're displaying but now that we're using schedules, it's not required.

File: app/services/participant_event_paid_aggregator.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (blackrat) [reply app/services/participant_event_paid_aggregator.rb]: It was used in the demo that I just worked through with Andy and Rhys, but I'll move it to the next PR too

**Pull request merged**